src/screens/ProfileScreen/index.js

```diff
@@ -68,9 +72,6 @@ import {setMyProfileFeed} from '../../context/actions/myProfileFeed';
 import {useAfterInteractions} from '../../hooks/useAfterInteractions';
 import {useUpdateClientGetstreamHook} from '../../utils/getstream/ClientGetStram';
 import {withInteractionsManaged} from '../../components/WithInteractionManaged';
-import ShadowFloatingButtons from '../../components/Button/ShadowFloatingButtons';
-import useCoreFeed from '../FeedScreen/hooks/useCoreFeed';
-import StorageUtils from '../../utils/storage';
 
 const {width} = Dimensions.get('screen');
 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import order has been changed here too. As mentioned before, this doesn't affect the functionality of the code, but it's good to keep related imports grouped together for better readability and maintainability.
```diff
- import ShadowFloatingButtons from '../../components/Button/ShadowFloatingButtons';
- import useCoreFeed from '../FeedScreen/hooks/useCoreFeed';
- import StorageUtils from '../../utils/storage';
+ import {useAfterInteractions} from '../../hooks/useAfterInteractions';
+ import {useUpdateClientGetstreamHook} from '../../utils/getstream/ClientGetStram';
+ import {withInteractionsManaged} from '../../components/WithInteractionManaged';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->